packages/esm-tb-app/src/views/patient-summary/tb-patient-summary.component.tsx

```diff
@@ -0,0 +1,24 @@
+import { EmptyStateComingSoon } from '@ohri/openmrs-esm-ohri-commons-lib';
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+interface OverviewListProps {
+  patientUuid: string;
+}
+
+const TBSummaryOverviewList: React.FC<OverviewListProps> = ({ patientUuid }) => {
+  const { t } = useTranslation();
+
+  const headerRecentTB = t('recentTuberculosis');
+  const headerPreviousCases = t('previousCases');
+  const headerVisit = t('visits');
```

Comment:
add default incase a translation doesn't exist for example we don't have translations for this in `Amaric`